bitnami/wordpress/values.yaml

```diff
@@ -1217,3 +1217,93 @@ externalCache:
+    ##
+    containerSecurityContext:
+      enabled: true
+      runAsUser: 1001
+      runAsNonRoot: true
+      privileged: false
+      readOnlyRootFilesystem: false
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: ["ALL"]
+      seccompProfile:
+        type: "RuntimeDefault"
+    ## @param cronJobs.scheduler.podAnnotations Additional pod annotations
+    ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+    ##
+    podAnnotations: {}
+    ## @param cronJobs.scheduler.podLabels Additional pod labels
+    ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+    ##
+    podLabels: {}
+    ## @param cronJobs.scheduler.failedJobsHistoryLimit The number of failed jobs to keep
+    ##
+    failedJobsHistoryLimit: 5
+    ## @param cronJobs.scheduler.successfulJobsHistoryLimit The number of successful jobs to keep
+    ##
+    successfulJobsHistoryLimit: 5
+    ## @param cronJobs.scheduler.concurrencyPolicy Specifies how to treat concurrent executions of a Job
+    ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#concurrency-policy
+    ##
+    concurrencyPolicy: Forbid
+    ## @param cronJobs.scheduler.backoffLimit Specifies the number of retries before marking this Job failed
+    ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#pod-backoff-failure-policy
+    ##
+    backoffLimit: 6
+    ## Additional init container to be added to the cronjob pod
+    ##
+    initContainer:
+      ## Curl image to check for readiness of main service
+      ## ref: https://hub.docker.com/r/curlimages/curl/tags/
+      ## @param cronJobs.scheduler.initContainer.image.registry [default: REGISTRY_NAME] Curl image registry
+      ## @param cronJobs.scheduler.initContainer.image.repository [default: REPOSITORY_NAME/os-shell] Curl image repository
+      ## @skip cronJobs.scheduler.initContainer.image.tag Curl image tag (immutable tags are recommended)
+      ## @param cronJobs.scheduler.initContainer.image.digest Curl image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+      ## @param cronJobs.scheduler.initContainer.image.pullPolicy Curl image pull policy
+      ## @param cronJobs.scheduler.initContainer.image.pullSecrets Curl image pull secrets
+      ##
+      image:
+        registry: docker.io
+        repository: curlimages/curl
+        tag: 8.5.0
+        digest: ""
+        pullPolicy: IfNotPresent
+        ## Optionally specify an array of imagePullSecrets.
+        ## Secrets must be manually created in the namespace.
+        ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+        ## e.g:
+        ## pullSecrets:
+        ##   - myRegistryKeySecretName
+        ##
+        pullSecrets: []
```

Comment:
It is not allowed to include non-Bitnami images on Bitnami charts. As an alternative you may use `bitnami/os-shell` image as other charts do.